src/app/pages/AvisoClientDetails/index.js

```diff
@@ -0,0 +1,222 @@
+import React, { Component } from 'react';
+import { Redirect } from 'react-router';
+import PaddedContainer from 'app/components/ui/PaddedContainer';
+import { CenteredContainer, ActionsContainer, ContainedButton } from './elements';
+import Text from 'app/components/ui/Text';
+import Icon from 'app/components/ui/Icon';
+import DotsStepper from 'app/components/ui/DotsStepper';
+import Spacer from 'app/components/ui/Spacer';
+import UserInfoListElement from 'app/components/ui/UserInfoListElement';
+import Checkbox from 'app/components/ui/Checkbox';
+//import Button from 'app/components/ui/Button';
+import { avisos, clients } from 'app/responses';
+
+class AvisoClientDetails extends Component{
+  
+  state = {
+    sameBilling: true
+  };
+  
+  render(){
+    const aviso = avisos.filter( (item) => item.id === parseInt(this.props.match.params.id, 10) )[0];
+    const client = typeof(aviso) !== 'undefined' ? clients.filter( (item) => item.id === parseInt(aviso.client, 10) )[0] : undefined;
```

Comment:
```suggestion
    const client = typeof(aviso) !== 'undefined'
       ? clients.filter( (item) => item.id === parseInt(aviso.client, 10) )[0]
       : undefined;
```
para que se lean mejor los cambios (si cambia).
Aunque porque `undefined`? no te sirve el `falsy`?
```suggestion
    const client = clients.filter( (item) => item.id === parseInt(aviso.client, 10) )[0];
```

Reply:
Es que tengo que verificar que aviso no es undefined porque dentro del map estoy haciendo aviso.client (que explota si es undefined)

Reply:
ok ok, entendi mal